### ISSUE TITLE
Calendar viz time issues when adding via dblclick

### ISSUE THREAD

**Issue by trob** [Fabrik/fabrik#1383]
setup (my timezone = MEZ = UTC + 1 (+2 with daylight saving))
1. date element UTC, no default
week view
dblclick: 9 Apr, 2:00 --> date in form: time = 1:00
dblclick: 24 Mar, 2:00 --> date in form: time = 1:00
day view
dblclick: 9 Apr, 2:00 --> date in form: time = 2:00
dblclick: 24 Mar, 2:00 --> date in form: time = 2:00

1. date element UTC, default to current date
week view
dblclick: 9 Apr, 2:00 --> date in form: time = 4:00
dblclick: 24 Mar, 2:00 --> date in form: time = 3:00
day view
dblclick: 9 Apr, 2:00 --> date in form: time = 3:00
dblclick: 24 Mar, 2:00 --> date in form: time = 2:00

So the only correct times are inserted in day view with date element with no default.

in removeDayEvents calendar.js is doing some timezone stuff, in removeWeekEvents it doesn't (which I assume is the reason for the 1h diff between day and week view time)
But no idea why a date with "default to current" is behaving so different (the default should be overridden with the dblclicked date/time anyway)

Some more combinations if date is set to local...

**Comment by pollen8** [comment]
@trob could you test this one please. I'm hoping its fixed

**Comment by trob** [comment]
no, seems the old calendar viz is doing it's own time calc before calling the form
Date UTC and default to current=YES: time is + 3 hours in day view and +1 hour in week view

Date UTC and default to current=NO:  +1 hour in day view, -1 hour in week view
